src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -84,7 +65,7 @@ public string ProblemSizeN
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(HPLinpackExecutor.ProblemSizeN), Environment.ProcessorCount * 10000);
+                return this.Parameters.GetValue<string>(nameof(this.ProblemSizeN), this.cpuInfo.LogicalCoreCount * 5000);
```

Comment:
I see you just changed it to *5000, and the document says *1000. How did you choose this number?